React/Fabric/Mounting/ComponentViews/TextInput/RCTTextInputUtils.h

```diff
@@ -41,4 +41,7 @@ UITextContentType RCTUITextContentTypeFromString(std::string const &contentType)
 API_AVAILABLE(ios(12.0))
 UITextInputPasswordRules *RCTUITextInputPasswordRulesFromString(std::string const &passwordRules);
 
+API_AVAILABLE(ios(11.0))
```

Comment:
I guess this is for the best but I don't think we even build for this iOS version :)

Reply:
Yeah, it's just to follow the pattern :)